src/pages/STG_PanelSchedule.page

```diff
@@ -108,6 +108,18 @@
                     </div>
                 </div>
             </div>
+
+            <div class="slds-grid slds-gutters_small">
+                <div class="slds-col slds-has-flexi-truncate slds-grid" role="listitem">
+                    <div class="slds-form-element">
+                        <apex:outputLabel value="{!$ObjectType.npo02__Households_Settings__c.Fields.Seasonal_Addresses_Batch_Size__c.Label}" for="seasonalAddressBatchSize" styleClass="slds-form-element__label" />
+                        <div class="slds-form-element__control">
+                            <apex:outputField value="{!stgService.stgHH.Seasonal_Addresses_Batch_Size__c}" rendered="{!isReadOnlyMode}" styleClass="slds-form-element__static" />
+                            <apex:inputField value="{!stgService.stgHH.Seasonal_Addresses_Batch_Size__c}" rendered="{!isEditMode}" id="seasonalAddressBatchSize" styleClass="slds-input" />
```

Comment:
- The first AC indicates "User can enter a batch size >0 for Seasonal Address on the NPSP Settings-->Batch Process Settings page". However, no other batch size on that page is validated on the input. @seanpat09 , can you please check if the input should be validated? Regardless of the input check, we still need to default the batch size to 10 if it is invalid since user can enter the value via Custom Settings page.
- If the batch size is null, should we default it to 10 when the batch size is displayed as either output or input?

Reply:
I agree that some level of validation needs to happen, but I think it's beyond the scope of this work item. We should be fine with the back end default for now in case the user inputs invalid values in this setting.

Reply:
Sounds good, thanks for verifying!